enterprise/server/backends/authdb/authdb.go

```diff
@@ -967,16 +967,19 @@ func (d *AuthDB) authorizeAPIKeyWrite(ctx context.Context, h interfaces.DB, apiK
 	if err != nil {
 		return nil, err
 	}
+	err = authutil.AuthorizeOrgAdmin(user, key.GroupID)
+	if err == nil {
+		// admins have write permissions on all keys in the group.
+		return key, nil
+	}
```

Comment:
`AuthorizeOrgAdmin` is not quite what we want here - we only want to allow API keys with ORG_ADMIN capability. See the authdb changes in https://github.com/buildbuddy-io/buildbuddy/pull/7666

Reply:
I was conflating admin role and admin capability. 🤦 

Will fix.